src/main/java/org/usfirst/frc4904/robot/commands/HoodZeroConstant.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Hood;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class HoodZeroConstant extends MotorConstant {
+    protected final boolean limitType;
+    protected final Hood hood;
+
+    public HoodZeroConstant(String name, Hood hood, double motorSpeed, boolean limitType) {
```

Comment:
1.) the motor speed should be a constant in this file
2.) please add to the javadoc a thing saying which limit positions correspond to `true` and `false` or just make the `limitType` an `enum`